modules/EventTracker.js

```diff
@@ -0,0 +1,187 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone', './ServiceCell'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'), require('./ServiceCell'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.EventTracker = factory(root._, root.Backbone, root.Torso.ServiceCell);
+  }
+}(this, function(_, Backbone, ServiceCell) {
+  'use strict';
+
+  var EventTracker = ServiceCell.extend({ 
```

Comment:
Trying to think if you triggered the event on the instance and on Torso.Events, would that cause issues involving multiple triggered callbacks? The events would be triggered in two different places. If not, it might give us the flexibility to do instance-level subscribing, and application-level subscribing.